Assets/Scripts/Services/Input/InputController.cs

```diff
@@ -0,0 +1,66 @@
+using System;
+using UnityEngine;
+using UnityEngine.InputSystem;
+using Zenject;
+
+namespace Services.Input
+{
+    public class InputController : IInputController
+    {
+        private readonly InputActionAsset _inputActionAsset;
+        
+        public event Action<Vector2> MoveChanged;
+        public event Action<Vector2> FireChanged;
+        public event Action<Vector2> SpecialChanged;
+        public event Action MeleeChanged;
+        public event Action UseChanged;
+        public event Action ReloadChanged;
+
+        [Inject]
+        public InputController(InputActionAsset inputActionAsset)
+        {
+            _inputActionAsset = inputActionAsset;
+            OnEnable();
+        }
+
+        private void Move(InputAction.CallbackContext ctx)
+        {
+            MoveChanged?.Invoke(Vector2.zero);
+        }
+
+        private void Fire(InputAction.CallbackContext ctx)
```

Comment:
Pls rename parametr to context. By our agreement, the use of abbreviations or acronyms must not be allowed. as an exception to this rule, we can take the name "UI" or "FX", which is currently recognized as a concept and not an acronym by the vast majority of people / programmers